|ERROR|2023-04-02 20:17:16,983|Internal Python error in the inspect module.
Below is the traceback from this internal error.

